1038-vertical-nav-design-guidance.md

```diff
@@ -0,0 +1,35 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-18
+updated: 2024-10-21
+---
+
+# Vertical Nav Component
+
+[Vertical Nav](https://clarity.design/documentation/vertical-nav) is a vertically aligned navigational component. It can fit as many navigation links as needed, scrolling when the content exceeds the viewport.
+
+
+## Guidance
+
+- **Must** use to collapse and expand the navigation to free up space for the content area.
+- **Must** use dividers and headers to separate logical clusters of navigation items.
+- **Must** NOT use vertical navigation with a header and a sub navigation menu. Having too many places for navigation will be confusing for users.
+
+## Nav Icons
+- **Must** maintain consistency across using icons on navigation items. Avoid using icons to some of top navigation items and not to others.
+- **Must** apply icons only to top-level navigation items. Do not add them to child links.
+- **Should** place icons to the left of the top navigation labels.
+- **Should** use icons to describe navigation items which are helpful when the navigation is collapsed.
+
+## Nav Hierarchy
+- **Must** use semi bold font weight for Top-level navigation items and have a caret to the right.
+- **Must** use normal font weight for child links.
+- **Must** use semi bold font weight Top-level navigation items without child item.
+
+## Content Guidelines
+- **Must** use short and consise label for navigation items inorder to prevent an ellipsis from showing up. Lengthy lables are trimmed and followed by an ellipsis.
```

Comment:
```suggestion
- **Must** use short and concise label for navigation items in order to prevent an ellipsis from being required. Lengthy labels are trimmed and followed by an ellipsis.
```